src/main/java/com/api/parkingmeter/application/usecase/UpdateParkingSession.java

```diff
@@ -6,19 +6,27 @@
 import lombok.RequiredArgsConstructor;
 import org.springframework.stereotype.Component;
 
+import java.math.BigDecimal;
+
 @Component
 @RequiredArgsConstructor
 public class UpdateParkingSession {
   private final ParkingSessionGateway parkingSessionGateway;
 
   public ParkingSession execute(final String licensePlate) {
-    final var parkingSession =
+    ParkingSession parkingSession =
         parkingSessionGateway
             .findActiveSessionByLicensePlate(licensePlate)
             .orElseThrow(() -> new ParkingSessionNotFoundException(licensePlate));
 
     parkingSession.setEndTime(parkingSession.getEndTime().plusHours(1));
 
+
+    BigDecimal additionalCost = BigDecimal.valueOf(10.00);
```

Comment:
vc pode trocar a linha 25 até a 28 por essa linha abaixo:
```suggestion
parkingSession.setTotalCost(parkingSession.getTotalCost().add(BigDecimal.valueOf(5)));
```